lib/membrane/rtp/vad/is_speaking_estimator.ex

```diff
@@ -0,0 +1,179 @@
+defmodule Membrane.RTP.Vad.IsSpeakingEstimator do
+  @moduledoc """
+  Module for estimating if the user is speaking inspired by
+  [*Dominant Speaker Identification for Multipoint Videoconferencing*](https://israelcohen.com/wp-content/uploads/2018/05/IEEEI2012_Volfin.pdf)
+  by Ilana Volfin and Israel Cohen
+
+  ------
+
+  The `estimate_is_speaking/2` function takes a list of audio levels in range [0, 127]
+  and based on a threshold given as a second input
+  computes if the person is speaking.
+
+  The input levels are interpreted on 3 tiers. Each tier consists of intervals specified below:
+
+  | Name         | Interpretation       | Number of RTP packets (default) | length    |
+  |--------------|----------------------|---------------------------------|-----------|
+  | `immediate`  | smallest sound chunk | 1                               | ~20 [ms]  |
+  | `medium`     | one word             | 1 \* 10 = 10                    | ~200 [ms] |
+  | `long`       | half/one sentence    | 1 \* 10 \* 7 = 70               | ~1.4 [s]  |
```

Comment:
This line is rendered incorrectly, although it seems like a bug in `ex_doc`
. Here's the fix:
```suggestion
  | `long`       | half/one sentence    | 1 \\* 10 \* 7 = 70              | ~1.4 [s]  |
```

You should probably also file a bug at [`ex_doc`](https://github.com/elixir-lang/ex_doc/issues) repo (it is possible the issue is deeper, in one the ex_doc deps but I'd start from the `ex_doc` team) with a minimal example of reproduction.